test/journeys/server/data-api/space.html

```diff
@@ -3,7 +3,36 @@
 <head>
   <title>Cisco Spark Widget Test</title>
   <meta charset="utf8">
-<link href="/dist/main.css" rel="stylesheet"></head>
+  <link href="/dist/main.css" rel="stylesheet">
+  <style type="text/css">
```

Comment:
I found that disabling all css animations in the journey tests helps with test stability!